### DISCUSSION TITLE
compatibility with quarkus 2.16

### DISCUSSION THREAD

**Discussion by luca-vercelli** [quarkiverse/quarkus-cxf#1548] [Q&A]
What version should I use with Quarkus 2.16?

**Comment by ppalaga** [comment]
We did not release for Quarkus 2.16. You may try with Quarkus CXF 1.7.3

**Comment by luca-vercelli** [comment]
Did you release some version for some Quarkus 2.x ?

Il giorno gio 10 ott 2024 alle ore 14:48 Peter Palaga <
***@***.***> ha scritto:

**Reply by ppalaga** [reply]
1.7.3 was tested and released against Quarkus 2.15 - see also the release table https://docs.quarkiverse.io/quarkus-cxf/dev/release-notes/index.html

**Comment by luca-vercelli** [comment]
Thank you.
